settings.gradle

```diff
@@ -0,0 +1,2 @@
+rootProject.name = "inspectit-oce-root"
```

Comment:
this should be `inspectit-oce` as that's what root directory will be called once you clone the git repo..

Reply:
In the old inspectIt rep owe named the root project "inspectit.root" even though the folder usually would be only "inspectit". I think the name of a Gradle-.Project is not necessarily coupled to the name of its folder. However I agree that it might avoid some confusion when naming it just "inspetit-oce".